### PR TITLE
CLN: FIXMEs

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/indexing/multiindex/test_partial.py [hunk 1]
```diff
@@ -169,28 +169,6 @@ def test_getitem_intkey_leading_level(
             with tm.assert_produces_warning(FutureWarning):
                 mi.get_value(ser, 14)
 
-    # ---------------------------------------------------------------------
-    # AMBIGUOUS CASES!
-
-    def test_partial_loc_missing(self, multiindex_year_month_day_dataframe_random_data):
```
Comment (jbrockmendel) [inline pandas/tests/indexing/multiindex/test_partial.py:175]: This was original an 'ix' test 10 years ago, never got enabled.  https://github.com/pandas-dev/pandas/blame/1f00335b51d345d58e9574c2c5d86613214aba9b/pandas/tests/test_multilevel.py#L1421

Comment (jreback) [reply pandas/tests/indexing/multiindex/test_partial.py:175]: not me, hahaha

Comment (jbrockmendel) [reply pandas/tests/indexing/multiindex/test_partial.py:175]: im proud of tracking this one down!

Comment (jreback) [reply pandas/tests/indexing/multiindex/test_partial.py:175]: heheh no doubt!

**Comment by jbrockmendel** [comment]
CI fail is from the lint issue fixed on master a little while ago

**Comment by jbrockmendel** [comment]
One that i'll need some help figuring out how to make actionable: https://github.com/pandas-dev/pandas/blob/master/pandas/plotting/_matplotlib/timeseries.py#L231

**Pull request merged**